src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -49,7 +50,18 @@ internal partial class InlineRenameSession : ForegroundThreadAffinitizedObject,
         /// If non-null, the current text of the replacement. Linked spans added will automatically be updated with this
         /// text.
         /// </summary>
-        public string ReplacementText { get; private set; }
+        public string ReplacementText
+        {
+            get
+            {
+                return _replacementText;
+            }
+            private set
+            {
+                _replacementText = value;
+                NewNameChanged?.Invoke(this, EventArgs.Empty);
```

Comment:
Why is the event you are raising named differently to the property you are changing?